zerver/models/users.py

```diff
@@ -917,8 +917,7 @@ def remote_user_to_email(remote_user: str) -> str:
 post_save.connect(flush_user_profile, sender=UserProfile)
 
 
-@cache_with_key(user_profile_by_id_cache_key, timeout=3600 * 24 * 7)
-def get_user_profile_by_id(user_profile_id: int) -> UserProfile:
+def _base_query_set() -> QuerySet[UserProfile]:
```

Comment:
This should use a name that is unique across the codebase, like what I proposed.

Reply:
Yes, you proposed `_base_queryset`, I added that extra `_` by mistake

Reply:
But for me `_base_user_queryset` is better as it's more relevant